auto_update_tests.py

```diff
@@ -184,6 +184,8 @@
            src, '-c', '-o', 'example.o',
            '-Isrc', '-g', '-L' + libdir, '-pthread']
   print 'build: ', ' '.join(extra)
+  if src.endswith('.cpp'):
+    extra += ['-std=c++11']
```

Comment:
I thought clang already defaulted to C++11?
(actually I think its gnu++11, but that doesn't matter, does it?)

Reply:
Depends on the version of clang and gcc. When I run locally in Ubuntu LTS using gcc, it needs that flag.